packages/cdktf/lib/terraform-operators.ts

```diff
@@ -0,0 +1,113 @@
+import { Expression, OperatorExpression } from "./tfExpression";
+import { IResolvable } from "./tokens/resolvable";
+
+/**
+ * This class contains static functions for all arithmetical and logical operators
+ * in the Terraform configutation language.
+ */
+export class Op {
+  /**
+   * Renders !expr
```

Comment:
nit: shall we match the name of the argument?
```suggestion
   * Renders !expression
```